### PR TITLE
feat: add grid component in docs

### PR DESCRIPTION
**Pull request by rohanmathur91**
## Description:
 - Added grid layout component in docs.
 
## Screenshots:
image
image

### CODE REVIEW

**Review by Creator-singh — APPROVED** [review]
great work

**Comment by arpit01923** [comment]
LGTM

File: components/grid.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.grid {
+	display: grid;
+}
+
+.grid-2 {
```
Comment (akshay-rajput) [inline components/grid.css]: Probably rename it in a more semantic way

File: components/grid.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+.grid-2 {
+	grid-template-columns: repeat(2, 1fr);
+}
+
+.grid-3 {
+	grid-template-columns: repeat(3, 1fr);
+}
+
+.grid-responsive {
```
Comment (akshay-rajput) [inline components/grid.css]: rename it to .grid-responsive-min-100

File: components/grid.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+}
+
+.grid-item {
```
Comment (akshay-rajput) [inline components/grid.css:18]: to make it reusable across projects, it should not be include other properties

File: documentation/index.html [hunk 4]
```diff
@@ -647,6 +650,100 @@ <h3>Jacket</h3>
                 </article>
             </section>
 
+
+            <!-- grid -->
+            <section id="grid">
+                <article>
+                    <div>
+                        <h2 class="font-black my-2">Grid</h2>
```
Comment (akshay-rajput) [inline documentation/index.html]: change to h3

**Review by akshay-rajput — CHANGES_REQUESTED** [review]
Make these minor changes.

File: components/grid.css (hunk 3 above)
Comment (rohanmathur91) [reply components/grid.css:18]: done

File: components/grid.css (hunk 2 above)
Comment (rohanmathur91) [reply components/grid.css]: done

File: components/grid.css (hunk 1 above)
Comment (rohanmathur91) [reply components/grid.css]: done

File: documentation/index.html (hunk 4 above)
Comment (rohanmathur91) [reply documentation/index.html]: done

**Pull request merged**